sbc_compassion/models/sponsorship_correspondence.py

```diff
@@ -79,7 +79,7 @@ class SponsorshipCorrespondence(models.Model):
         readonly=True)
     # First spoken lang of partner
     original_language_id = fields.Many2one(
-        'res.lang.compassion', compute='_set_languages',
+        'res.lang.compassion', readonly=True,
```

Comment:
What if someone enters manually a letter ? Will he be able to select the language ?